src/sinks/influxdb/logs.rs

```diff
@@ -163,7 +164,12 @@ impl HttpSink for InfluxDBLogsSink {
         let mut event = event.into_log();
 
         // Measurement
-        let measurement = encode_namespace(Some(&self.namespace), '.', "vector");
+        let name = "vector";
+        let measurement = encode_namespace(
+            Some(self.namespace.as_str()).filter(|namespace| !namespace.is_empty()),
```

Comment:
I could see moving this check for empty string into `encode_namespace` as I think we'd want it everywhere.

Reply:
It was used everywhere, for `namespace` option, but with `default_namespace` our goal is to enforce having a namespace so we don't want it there.